src/riak_kv_wm_index.erl

```diff
@@ -584,11 +584,13 @@ otp_encode_results(false, Results, Continuation) ->
     ).
 
 results_encode({Term, Key}, Encode) when is_binary(Term), is_binary(Key) ->
-    ["{", [Encode(Term, Encode), $: | Encode(Key, Encode)], "}"];
+    [123, [Encode(Term, Encode), $: | Encode(Key, Encode)], 125];
```

Comment:
would you not rather use `${`, since we don't all have the ascii table in our head? And you already do it for `:`.
```suggestion
    [${, [Encode(Term, Encode), $: | Encode(Key, Encode)], $}];
```